core/src/components/datetime/datetime.tsx

```diff
@@ -93,6 +93,12 @@ export class Datetime implements ComponentInterface {
   private minParts?: any;
   private maxParts?: any;
 
+  /**
+   * Duplicate reference to `activeParts` that does not rigger a re-render of the component.
```

Comment:
```suggestion
   * Duplicate reference to `activeParts` that does not trigger a re-render of the component.
```

Let's also add an explanation of when/why this is used